pkg/utils/utils.go

```diff
@@ -284,6 +284,44 @@ func DirExists(path string) bool {
 	return err == nil
 }
 
+// IsSymlinkValidAndExists will check whether the symlink exists and points to a valid target
+// return three values: whether the symlink exists, whether it points to a valid target, and any error encountered
+func IsSymlinkValidAndExists(symlinkPath string) (bool, bool, error) {
```

Comment:
Add unit test cases for the windows platform.

Reply:
`IsSymlinkValidAndExists` is only used on unix-like system, I added some comments for this.